api/base/serializers.py

```diff
@@ -877,9 +877,9 @@ def to_representation(self, value):
             related_class = resolved_url.func.view_class
             if issubclass(related_class, RetrieveModelMixin):
                 try:
-                    related_type = resolved_url.namespace
+                    related_type = resolved_url.namespace.split(':')[-1]
```

Comment:
It is `providers:preprint-providers` or `providers:registration-providers`. It perhaps wasn't necessary before because we used the now deprecating `preprint_providers` namespace.

Reply:
although perhaps that means the later line 
`if related_type in ('preprint-providers', 'registration-providers'):` 
should be 
`if related_type in ('preprint_providers', 'preprint-providers', 'registration-providers'):`
to ensure compatibility with anything still using the older namespace/api endpoints.